.stylelintrc.js

```diff
@@ -7,5 +7,15 @@ module.exports = {
     // This rule can lead to less readable code
     // @see: https://github.com/stylelint/stylelint-config-recommended/issues/14
     'declaration-block-no-redundant-longhand-properties': null,
+    // Allow UNSTABLE_Component classnames
+    // @see: https://github.com/lmc-eu/spirit-design-system/blob/main/docs/contributtion/experimental-code.md
+    'selector-class-pattern': [
+      '(^(UNSTABLE_)?([A-Z][a-zA-Z0-9]*)((--|__)[a-z][a-zA-Z0-9]*)*$)|(^([a-z][a-z0-9]*)(-[a-z0-9]+)*$)',
+      {
+        message:
+          'Expected class selector to be in format `MyComponent__myElement`, `MyComponent--modifier` ' +
```

Comment:
Is it possible to use a multiline template string, please, e. g. without the `+` string concatenation?